src/isIsogram.test.js

```diff
@@ -7,13 +7,25 @@ describe(`Function 'isIsogram':`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
 
-  it(`should return a boolean`, () => {
+  it(`should return a boolean result`, () => {
+    expect(typeof isIsogram('boolean')).toBe('boolean');
+  });
 
+  it(`shouldn't be case sensitive`, () => {
+    expect(isIsogram('Hi')).toBe(true);
   });
```

Comment:
Here you don't check case sensitivity because you don't have the same letter in both cases. Better use the input `'Wow'` for it (from the fifth test)